src/shared/components/ProfessionCard.tsx

```diff
@@ -21,28 +20,26 @@ const CardHeaderContainer = styled(CardHeader)`
 const AvatarContainer = styled(Avatar)<{ background: ProfessionKey }>`
   margin-right: 1em;
   background-color: ${({ background, theme }) => theme.palette.professions[background].main};
-  padding: .75em;
+  padding: 0.35em;
 `;
 
-type CardProps = {
+type ProfessionCardProps = {
   title: string;
   description: string;
   avatarSrc: string;
   avatarBackground: ProfessionKey;
-  clickHandler: (route: string) => void;
-  route: string;
+  isClickable?: boolean;
 };
 
-export const ProfessionCard: React.FC<CardProps> = ({
+export const ProfessionCard: React.FC<ProfessionCardProps> = ({
   title,
   description,
   avatarSrc,
   avatarBackground,
-  clickHandler,
-  route,
+  isClickable,
 }) => (
-  <CategoryCardContainer onClick={() => clickHandler(route)}>
+  <ProfessionCardContainer $isClickable={isClickable !== undefined}>
```

Comment:
If we make `isClickable` a non-optional prop, we can remove this `!==` :)

Reply:
When we set isClickable as required prop we have to remember about it in every component. With optional isClickable we can add it only with component with 'clickable style'. That was an idea, why isClickable is optional :)

Reply:
You can use a default props in here `export const ProfessionCard: React.FC<ProfessionCardProps> = ({
  title,
  description,
  avatarSrc,
  avatarBackground,
  isClickable = false,
})`
This way we can get rid of this `undefined` and we don't have to add `isClickable` each time.

Reply:
@klaraborowska I used your advice but when I set `isClickable` as required in type `ProfessionCard` typescript still complains about missing props if I don't provide it explicitly (even if I set default props).
I join your hints - and `isClickable` is an optional prop with default value `false`.

Reply:
Maybe I didn't write it clear enough, but you did exactly what I meant ;-)